indico/modules/events/registration/forms.py

```diff
@@ -357,6 +357,10 @@ class BadgeSettingsForm(IndicoForm):
                                  description=_("Display a dashed border around each badge"))
     page_size = IndicoEnumSelectField(_('Page size'), enum=PageSize)
     page_orientation = IndicoEnumSelectField(_('Page orientation'), enum=PageOrientation)
+    page_layout = IndicoEnumSelectField(_('Page layout'), enum=PageLayout,
+                                        description=_('Single sided (foldable) option is only available if the '
+                                                      'template orientation is the same as the page orientation and its'
+                                                      ' width is exactly half of the page width'))
```

Comment:
Put the initial space at the end of the previous line, similarly to the first line of the description.